src/coprocessor/xeval/evaluator.rs

```diff
@@ -521,6 +521,11 @@ impl Evaluator {
     fn eval_scalar_function(&mut self, ctx: &EvalContext, expr: &Expr) -> Result<Datum> {
         match expr.get_sig() {
             ScalarFuncSig::AbsInt => self.abs_int(ctx, expr),
+            ScalarFuncSig::AbsReal => self.abs_real(ctx, expr),
+            ScalarFuncSig::CeilInt => self.ceil_int(ctx, expr),
```

Comment:
In TiDB, we support both ceil and ceiling, 
they totally the same behavior. @hhkbp2